### PR TITLE
Add translation for ReactDOMServer

### PR DESCRIPTION
**Pull request by Manugal**

### CODE REVIEW

**Comment by LucaBlackDragon** [comment]
Hi @Manugal, is this PR ready for review?
Could you please set up reviewers and labels?
Look at the other PRs for reference.
Thanks!

**Comment by Manugal** [comment]
Hi.
Yes, it's ready. I referenced this PR right now. I can't set up reviewers and labels by myself, maybe lack of permissions.

File: content/docs/reference-react-dom-server.md [hunk 1]
```diff
@@ -61,15 +62,15 @@ If you plan to use React on the client to make the markup interactive, do not us
...
+Se invochi [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) su un nodo che già possiede questo markup renderizzato lato server, React lo preserverà e aggancierà solamente i gestori degli eventi, consentendoti di avere un caricamento iniziale molto performante.
 
-> Note:
+> Nota:
 >
-> Server-only. This API is not available in the browser.
+> Solo server. Questa API non è disponibile nel browser.
 >
-> The stream returned from this method will return a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
+> Lo stream ritornato da questo metodo sarà uno stream di bytes codificato in utf-8. Se hai bisogno di uno stream in un'altra codifica, da uno sguardo a un progetto tipo [iconv-lite](https://www.npmjs.com/package/iconv-lite), il quale fornisce stream di transformazione per la transcodifica del testo.
```
Comment (deblasis) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> Lo stream ritornato da questo metodo sarà uno stream di bytes codificato in utf-8. Se hai bisogno di uno stream in un'altra codifica, da uno sguardo a un progetto tipo [iconv-lite](https://www.npmjs.com/package/iconv-lite), il quale fornisce stream di trasformazione per la transcodifica del testo.
```

File: content/docs/reference-react-dom-server.md [hunk 2]
```diff
@@ -79,14 +80,14 @@ If you call [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on a node that
...
+Se hai intenzione di usare React sul client per creare un markup interattivo, non usare questo metodo. Utilizza, invece, [`renderToString`](#rendertostring) sul server e [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) sul client.
 
-> Note:
+> Nota:
 >
-> Server-only. This API is not available in the browser.
+> Solo server. Questa API non è disponibile nel browser.
 >
-> The stream returned from this method will return a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
+> Lo stream ritornato da questo metodo sarà uno stream di bytes codificato in utf-8. Se hai bisogno di uno stream in un'altra codifica, da uno sguardo a un progetto tipo [iconv-lite](https://www.npmjs.com/package/iconv-lite), il quale fornisce stream di transformazione per la transcodifica del testo.
```
Comment (deblasis) [inline content/docs/reference-react-dom-server.md]:
```suggestion
> Lo stream ritornato da questo metodo sarà uno stream di bytes codificato in utf-8. Se hai bisogno di uno stream in un'altra codifica, da uno sguardo a un progetto tipo [iconv-lite](https://www.npmjs.com/package/iconv-lite), il quale fornisce stream di trasformazione per la transcodifica del testo.
```

File: content/docs/reference-react-dom-server.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ category: Reference
 permalink: docs/react-dom-server.html
 ---
 
-The `ReactDOMServer` object enables you to render components to static markup. Typically, it's used on a Node server:
+L'oggetto `ReactDOMServer` ti permette di renderizzare componenti in markup statici. Tipicamente viene usato su un server Node:
```
Comment (deblasis) [inline content/docs/reference-react-dom-server.md]:
```suggestion
L'oggetto `ReactDOMServer` ti permette di renderizzare componenti in markup statici. Tipicamente viene usato con un server Node:
```
I feel a bit battled about this. Technically you use it "on top of" as in: `su` but wouldn't it be better to say `con`? Leave it if you disagree

Comment (Manugal) [reply content/docs/reference-react-dom-server.md]: Yes, I agree with you. The word 'con' is more appropriate in this context.

**Pull request merged**